.requirements

```diff
@@ -7,6 +7,7 @@ RESTY_LUAROCKS_VERSION=3.8.0
 RESTY_OPENSSL_VERSION=1.1.1n
 RESTY_PCRE_VERSION=8.45
 RESTY_LMDB_VERSION=master
+RESTY_EVENTS_VERSION=dev
```

Comment:
Same: we'll have to make a resty events release and change this before merging.

Reply:
now it is `main`